content/en/blog/2024/demo-skyscanner/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title:
+  'Making observability fun: How we increased engineers confidence in incident
+  management using a game'
+linkTitle: Skyscanner using OtelDemo
+date: 2024-02-22
+author: >-
+  [Jordi Bisbal Ansaldo](https://github.com/jordibisbal8) (Skyscanner)
+---
+
+At Skyscanner, as in many organisations, teams tend to follow specific runbooks
+for individual failure modes. With modern and complex distributed systems, this
+has the downside of most of the errors being unknowns, which makes runbooks
+practically useless.
+
+After migrating our telemetry data to the OpenTelemetry standards, at
+Skyscanner, we now have richer instrumentation and can rely on observability
+directly. As a result, we are ready to adopt a new
+[observability mindset](https://charity.wtf/2019/09/20/love-and-alerting-in-the-time-of-cholera-and-observability/),
+which requires training our engineers to work effectively with the new
+ecosystem. This allows them to react efficiently to any known or unknown issues,
+even under pressure.
+
+To achieve this, we believe that the best way to gain knowledge isn’t through
+one-time viewings of documents or videos. Instead, it’s through practical
+exercises that include situations with never-before-seen (or at least rarely
+seen) problems. This helps the company reduce the time to mitigate an issue
+(TTM), which goes from when a first responder acknowledges the incident to the
+time users stop suffering from the incident.
+
+## Environment
+
+To begin with, we need to set up an environment that demonstrates the best
+practices for monitoring and debugging using OpenTelemetry instrumentation and
+observability. For this, we propose the use of the official
+[OpenTelemetry Demo project](https://github.com/open-telemetry/opentelemetry-demo),
+which is a realistic example of a distributed system called Astronomy Shop.
+Thanks to its OpenTelemetry Protocol (OTLP), it allows us to simply point the
+standard OTLP exporter in the Collector to [New Relic](https://newrelic.com/),
+our chosen observability platform at Skyscanner which, like other platforms, is
+fully embracing open standards to ingest telemetry data.
```

Comment:
@open-telemetry/docs-approvers double checking if we are OK with naming an observability vendor or if this should be "to our chosen observability platform". I am OK with it since it's an end-user blog post.

Reply:
This is still a valid point (although outdated). Considering the trace screenshot below comes from New Relic, it makes me think it's okay to mention it here. I don't think the blogpost is trying to advertise New Relic features except OTLP ingest, and in any case it compares it to other platforms that also accept OTLP.